sdk/include/psdk/shellapi.h

```diff
@@ -618,6 +618,12 @@ SHEmptyRecycleBinW(
   _In_opt_ LPCWSTR pszRootPath,
   DWORD dwFlags);
 
+BOOL
+WINAPI
+SHRunControlPanel(
+  _In_ LPCWSTR lpcszCmdLine,
+  _In_opt_ HWND hwndMsgParent);
```

Comment:
Again, please move that in shlobj.h (Otherwise this breaks compatibility of our psdk with MS one).

Reply:
Ongoing !